v0.4.0/Azure-NDv4-A100-quickscan/baseline.json

```diff
@@ -0,0 +1,32 @@
+{
+  "gemm-flops/bf16_tc_flops": 265900.55,
+  "gemm-flops/fp16_flops": 33858.0635,
+  "gemm-flops/fp16_tc_flops": 280435.1458,
+  "gemm-flops/fp32_flops": 18353.002,
+  "gemm-flops/fp64_flops": 9030.9323,
+  "gemm-flops/fp64_tc_flops": 18947.2453,
+  "gemm-flops/int4_tc_iops": 975854.4755,
+  "gemm-flops/int8_tc_iops": 473072.78125,
+  "gemm-flops/return_code": 0,
+  "gemm-flops/tf32_tc_flops": 128225.0141,
+  "ib-loopback/ib_write_8388608_ib0_bw": 23.68471956,
+  "ib-loopback/ib_write_8388608_ib1_bw": 23.68471956,
```

Comment:
do you need to merge ib* into one metric?

Reply:
For baseline, we cannot support this feature now.